src/Slick/Pandoc.hs

```diff
@@ -59,8 +59,11 @@ markdownToHTML' :: (FromJSON a)
                 -> WriterOptions  -- ^ Pandoc writer options to modify output
                 -> T.Text         -- ^ Text for conversion
                 -> Action a
-markdownToHTML' rops wops txt =
-  mseq (markdownToHTML rops wops) convert txt
+markdownToHTML' rops wops =
+  (markdownToHTML rops wops) >=> convert
```

Comment:
You may remove superfluous parenthesis, since function application has always higher priority

Reply:
removed